flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientITCase.java

```diff
@@ -155,6 +155,19 @@ public void testSqlStatements() throws IOException {
     private static final String ERROR_BEGIN = "\u001B[31;1m";
     private static final String ERROR_END = "\u001B[0m";
 
+    private static String getInputFromPath(String sqlPath) throws IOException {
+        URL url = CliClientITCase.class.getResource("/" + sqlPath);
+        String in = IOUtils.toString(url, StandardCharsets.UTF_8);
+        // replace the placeholder with specified value if exists
+        return StringUtils.replaceEach(
+                in,
+                new String[] {"$VAR_PIPELINE_JARS", "$VAR_REST_PORT"},
+                new String[] {
+                    udfDependency.toString(),
+                    miniClusterResource.getClientConfiguration().get(PORT).toString()
```

Comment:
I think the replace variables will be more in the future, maintian them in two separate list is not good. 
Personally, I perfer to have a private static `Map<String, String> REPLACE_VARS` and initialize the map in `setup` method.